components/I18n/I18n.php

```diff
@@ -115,37 +150,30 @@ public function init() {
 			 * LABEL REPLACEMENT.
 			 */
 
-			// Setting pages.
+			// Menu labels.
 			add_filter( 'pods_admin_menu_page_title', array( $this, 'translate_admin_menu_page_title' ), 10, 2 );
 			add_filter( 'pods_admin_menu_label', array( $this, 'translate_admin_menu_label' ), 10, 2 );
 
-			// Pod Objects.
-			add_filter( 'pods_whatsit_get_label', array( $this, 'translate_label' ), 10, 2 );
-			add_filter( 'pods_whatsit_get_description', array( $this, 'translate_description' ), 10, 2 );
-
-			foreach ( pods_form()->field_types() as $type => $data ) {
-				add_filter(
-					'pods_form_ui_field_' . $type . '_options',
-					array( $this, 'translate_field_options' ),
-					10,
-					5
-				);
+			// Do not overwrite Whatsit object fields if we're editing Pods.
+			if ( ! $is_pods_edit_page && ! pods_doing_json() && ! wp_doing_ajax() ) {
```

Comment:
@sc0ttkclark 
Can you think of other areas where Whatsit translations should not occur?
Also, is `! pods_doing_json() && ! wp_doing_ajax()` the best way or is there a better way to tackle these requests since 2.8?

Reply:
Still the best way.